wordpress-mobile/WordPress-Android
# [Gutenberg] updated GB-mobile hash to fix rotation problems on Android

Fixes a problem with rotation on Android with empty Paragraphs

Gutenberg PR https://github.com/WordPress/gutenberg/pull/14563
Mobile Gutenberg PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/770

To test:
1. start a draft
2. do not enter any text
3. rotate the screen
4. observe it doesn't crash anymore (would appear a red screen on debug mode)

Update release notes:
